### ISSUE TITLE
Values out of bounds

### ISSUE THREAD

**Issue by oskkark** [DSSAT/pythia2ggcmi#6]
"Values are out of bounds. cnyield has Inf values, harvest years can reach 2101, biomass has small negative numbers, ..."

**Comment by frostbytten** [comment]
Do we have a list of all the out of bounds values?

**Comment by oskkark** [comment]
> Do we have a list of all the out of bounds values?

Yes, here is the report file with rows labeled as "outside valid range".
[DSSAT-Pythia_data_issues.txt](https://github.com/DSSAT/pythia2ggcmi/files/7804503/DSSAT-Pythia_data_issues.txt)
